docs/api/dataset.rst

```diff
@@ -0,0 +1,99 @@
+Datasets
+========
+
+General datasets
```

Comment:
I find the `dataset(s)` part of subsection titles to be redundant; could you remove them?